outline.Rmd

```diff
@@ -23,439 +21,671 @@ if (!file.exists ("statsoft.bib")) {
+Defining "Statistical Software"
+-------------------------------
 
-There is no ready definition for "statistical software", but nor may such
-a definition be essential to the success of the present project. An
+There is no ready definition for "statistical software", but nor may
+such a definition be essential to the success of the present project. An
 [analysis](https://github.com/mpadge/statistical-software/tree/master/jss)
-conducted in preparing this document of all historical submissions to the
-Journal of Statistical Software revealed a notable *homogeneity* in textual
-descriptions of software, with no notable phrases or topics which might be
-useful to define or categorise statistical software, other than topics
-pertaining to particular areas of application. That journal defines [its
-own scope](https://www.jstatsoft.org/pages/view/mission) as, 
-
-> statistical computing in all areas of empirical research.
-
-It may suffice, at least initially, to simply provide broad definitions of what
-might lie *beyond* the scope of the present project, or even to assert that
-statistical software must at heart implement some kind of statistical algorithm
-(or perhaps *employ* or otherwise interface with such in sufficiently novel
-form). This is nevertheless a categorical exercise, and so one way or another we anticipate having to address the fundamental quesion(s) of:
-
-1. What categories of statistical software might be considered *in scope*?
-2. What categories of statistical software might be considered *out of scope*?
-
-More specific questions of scope which will also need to be addressed include
-the potential acceptability of the following different external forms of
-software:
-
-  - R packages
-  - Other forms of bundling or presenting R code other than standard packages
-  - Formats appropriate for other languages (such as python) yet with some
-    interface with the R language
-  - Independently-developed software in different languages which the
-    submission exposes to the R environment ("wrapper" packages)
+conducted in preparing this document of all historical submissions to
+the Journal of Statistical Software revealed a notable *homogeneity* in
+textual descriptions of software, with no notable phrases or topics
+which might be useful to define or categorise statistical software,
+other than topics pertaining to particular areas of application. That
+journal defines [its own
+scope](https://www.jstatsoft.org/pages/view/mission) as,
 
-It considering both of these issues of definition, it will be important to
-consider whether it may be advantageous or necessary to develop different
-procedures for different categories, whether in terms of categories of external
-form or categories of statistical software.
+statistical computing in all areas of empirical research.
 
+-   Supervised / regression models and algorithms
 
----
+-   Unsupervised models and algorithms
```

Comment:
Where are these list items from? They are not part of JSS scope, so you could please add a contextualizing sentence for them?

Reply:
I didn't get to it to finishing restructuring this section but will. I was thinking something like:

- The other axis of scope is what constitutes "statistical software," which could potentially encompass many things, including but not limited to,
- [bulleted list]
- JSS defines it this way [others define it some other way?]
- Defining this empirically is difficult - see text analysis appendix.